### PR TITLE
Perl bio phylo cipres

### PR DESCRIPTION
**Pull request by rvosa**
Bio::Phylo::CIPRES is a Perl (CPAN) library and thin command-line wrapper for interacting with the RESTful interface of the CyberInfrastructure for Phylogenetic RESearch (phylo.org). With it, users can submit analyses to the HPC portal and fetch their results, either within larger Perl programs or as part of linux-based workflows.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by rvosa** [comment]
Hi @abretaud, @bgruening previously you were kind enough to package Bio::Phylo for bioconda (thanks! That was awesome). I'm trying to fly solo on this one but I'm a little sketchy on the entire workflow. Am I doing this right? Is there anything more I should be doing, or just sit tight and wait for the review to come in?

**Comment by rvosa** [comment]
@BiocondaBot please add label

File: recipes/perl-bio-phylo-cipres/v0.2.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "perl-bio-phylo-cipres" %}
+{% set version = "v0.2.1" %}
+{% set sha256 = "d178a53426a76c3e2205d8caf02463a1ed28a814c45d98a4ac1ac98c90d9d920" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/R/RV/RVOSA/Bio-Phylo-CIPRES-v0.2.1.tar.gz
```
Comment (bgruening) [inline recipes/perl-bio-phylo-cipres/v0.2.1/meta.yaml:10]: you can use here the `{{ version }}` as well ...

**Comment by rvosa** [comment]
@bioconda-bot please merge

**Pull request merged**